README.md

```diff
@@ -10,6 +10,23 @@ You can install the command globally using:
 npm i -g @projectsophon/df-plugin-dev-server
 ```
 
+## Docker image installation and running
+
+You need to create network: `docker network create webproxy`
+
+From main directory you can execute commands to build and start docker image:
+
+```bash
+make up -> starts docker image
+make up-build -> creates docker image and starts it
+make restart -> restarts docker image currently running
+make stop -> stops docker container
+make down -> stops docker container and removes image
+make build -> creates docker image
```

Comment:
Let's drop the makefile and just tell people what docker commands to run

Reply:
Ok